query/src/sessions/session_settings.rs

```diff
@@ -137,6 +137,34 @@ impl Settings {
                 level: ScopeLevel::Session,
                 desc: "Enable new processor framework if value != 0, default value: 1",
             },
+
+            SettingValue {
```

Comment:
Why we add these parameters which only used by CSV  to the session settings?

Reply:
1. With this, users can specify their default format styles in settings. 
2. We can build `FormatSettings` from `QueryContext`, all serialization methods have `FormatSettings` args as [ClickHouse did](https://github.com/ClickHouse/ClickHouse/blob/ed424372199cc424fbe51672688796a58ec6d30c/src/DataTypes/Serializations/SimpleTextSerialization.h#L9-L60)

In the future, we can have a custom input/output format with settings.

For example:

```
echo "select * from ontime format CSV" | mysql -uroot --field_delimiter '\t' 

echo "insert into ontime format TabSeparated" |  mysql -uroot --field_delimiter ','     < ~/a.csv
```

Refer to [ClickHouse settings](https://github.com/ClickHouse/ClickHouse/blob/2e0d3efba0e22748507f204df444b72846849a8e/src/Core/Settings.h)